app/code/Magento/InventoryExportStockApi/Api/Data/ProductStockDataInterface.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryExportStockApi\Api\Data;
+
+/**
+ * Interface ProductStockDataInterface
+ */
+interface ProductStockDataInterface
+{
+    public const SKU = 'sku';
+    public const QTY = 'qty';
+
+    /**
+     * Sets SKU
+     *
+     * @param string $sku
+     * @return void
+     */
+    public function setSku(string $sku): void;
+
+    /**
+     * Provides SKU
+     *
+     * @return string
+     */
+    public function getSku(): string;
```

Comment:
I expect that we need to preserve nullable response so that the return type supposed to be `?string`